### PR TITLE
Update content in generating-ballerina-code-for-protocol-buffer-definitions.md

### PR DESCRIPTION
**Pull request by suleka96**
## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The code generation tool can produce `ballerina stub` and `ballerina service/cli
 You can generate Ballerina source code using the following command:
 
 ```
-./ballerina grpc --input <proto-file-path> [--output <path>] [--mode client | service]
+$ bal grpc --input <proto-file-path> [--output <path>] [--mode client | service]
```
Comment (sanjana) [inline swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md:28]:
```suggestion
$ bal grpc --input <proto-file-path> [--output <path>] [--mode client | service | proxy]
```
The output of the `bal help grpc` command is as following

image

**Review by sanjana — CHANGES_REQUESTED** [review]

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md (hunk 1 above)
Comment (suleka96) [reply swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md]: Fixed

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md [hunk 2]
```diff
@@ -38,13 +38,12 @@ If the output path is not specified, the output will be written to a directory c
...
 
-`--mode`   - Set the mode (client or service) to generate code samples. If not specified, only the stub file is
- generated.
+`--mode`   - Set the mode as client or service to generate code samples. Set the mode as proxy to generate a gateway proxy. If not specified, only the stub file is generated.
 
 
 ## Sample
 
-The below example shows how you can generate Ballerina source code from the following Protocol Buffers definition (in the `helloworld_service.proto` file).
+The below example shows how you can generate Ballerina source code from the following Protocol Buffer's definition (in the `helloworld_service.proto` file).
```
Comment (sanjana) [inline swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md:46]: Wouldn't it be much better to use either `Protocol Buffers' definition` or [`Protocol Buffers definition`](https://developers.google.com/protocol-buffers)? 

@praneesha WDYT?

**Review by sanjana — APPROVED** [review]
LGTM

**Review by sanjana — APPROVED** [review]

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md (hunk 2 above)
Comment (praneesha) [reply swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md]: Should we say "Protocol Buffer definition" @daneshk?

Comment (daneshk) [reply swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md]: `Protocol Buffers` is the correct name. Shall we keep it as it is?

Comment (praneesha) [reply swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md]: +1 Thanks a lot!

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ The code generation tool can produce `ballerina stub` and `ballerina service/cli
 You can generate Ballerina source code using the following command:
 
 ```
-./ballerina grpc --input <proto-file-path> [--output <path>] [--mode client | service]
+$ bal grpc --input <proto-file-path> [--output <path>] [--mode client | service | proxy]
```
[the remark this answers is not in the document]
Comment (daneshk) [reply swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md:28]: Let's remove the proxy command. we are currently not supporting it for sl

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md [hunk 4]
```diff
@@ -38,13 +38,12 @@ If the output path is not specified, the output will be written to a directory c
  Buffers definition. 
 If the package is not specified, the output will be written to a 'temp' directory in the current location.
 
-`--mode`   - Set the mode (client or service) to generate code samples. If not specified, only the stub file is
- generated.
+`--mode`   - Set the mode as client or service to generate code samples. Set the mode as proxy to generate a gateway proxy. If not specified, only the stub file is generated.
```
Comment (daneshk) [inline swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md:41]:
```suggestion
`--mode`   - Set the mode as client or service to generate code samples. If not specified, only the stub file is generated.
```

File: swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md [hunk 5]
```diff
@@ -38,13 +38,12 @@ If the output path is not specified, the output will be written to a directory c
...
 
 
 ## Sample
 
-The below example shows how you can generate Ballerina source code from the following Protocol Buffers definition (in the `helloworld_service.proto` file).
+The below example shows how you can generate Ballerina source code from the following Protocol Buffers' definition (in the `helloworld_service.proto` file).
```
Comment (daneshk) [inline swan-lake/learn/generating-ballerina-code-for-protocol-buffer-definitions.md:46]:
```suggestion
The below example shows how you can generate Ballerina source code from the following Protocol Buffers definition (in the `helloworld_service.proto` file).
```

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**